Cannot find module './models/Screenshot'

Hi, today i give the next error in my proyect using version 3.2.2 and 3.2.1. 

> backend_1  | timestamp=2022-02-09-16:53:10-0300 app=app id=0 type=error message=Error: Cannot find module './models/Screenshot'
> backend_1  | timestamp=2022-02-09-16:53:10-0300 app=app id=0 type=error message=Require stack:
> backend_1  | timestamp=2022-02-09-16:53:10-0300 app=app id=0 type=error message=- /app/node_modules/node-html-to-image/dist/main.js

I solve it using 3.2.1 in my package.json
`"node-html-to-image": "3.2.0",`

Hi, today i have got the same issus, when i build my typescript project and startting to run the server it's said

, error: Cannot find module './models/Screenshot'

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "forceConsistentCasingInFileNames": true,
     "module": "commonjs",
     "noImplicitAny": true,
     "rootDir": "./src",

